requirements.txt

```diff
@@ -0,0 +1,2 @@
+Django==2.0
+django-govuk-template
```

Comment:
We should always specify a version - the latest version is currently `0.7`.

Reply:
No problem. Updated to be more specific in commit `b58aca6`.